com.reprezen.swagedit.dependencies/pom.xml

```diff
@@ -32,8 +32,33 @@
 		</dependency>
 	</dependencies>
 
+	<repositories>
+		<repository>
+			<id>sonatype-snapshots</id>
```

Comment:
It looks like we are adding the sonatype repo so that dev builds of KZOE can pick up snapshot releases of KZOP. And this is needed now, I assume, because we're no longer using an update site for KZOP. Is that right?

Reply:
yes